src/current/cockroachcloud/cluster-management.md

```diff
@@ -39,12 +39,11 @@ To view and manage a specific cluster, click the name of the cluster. The [**Ove
 
 The [**Overview** page](cluster-overview-page.html?filter=dedicated) displays details about the selected {{ site.data.products.db }} cluster:
 
-- The **Current Charges** and next billing date for the cluster
-- The cluster's **Cloud** provider
-- The cluster's **Hardware per node**
-- A list of the selected cluster's nodes.
+- The cluster's **Configuration**, which includes the cloud provider, plan type, region(s), nodes and their status, compute per node, and storage per node.
 
-    For each node, the page displays the node's `Name` and `Status`, nested under its region.
+- The **Cluster upgrade** settings, which include the upgrade window, any upcoming upgrades, and the upgrade exclusion period.
```

Comment:
I changed the title of the card to **Cluster upgrades** (in [#13304](https://github.com/cockroachlabs/managed-service/pull/13304)) and the only fields that made the final cut are,
- upgrade window
- delay patch upgrades